.taskmaster/agent-orchestrator.sh

```
@@ -387,6 +401,34 @@ case "${1:-}" in
             echo "${agent}: ${status}"
         done
         ;;
+    "restart")
+        # Restart a specific agent
+        if [ -z "$2" ]; then
+            error "Usage: $0 restart <agent-id>"
+            exit 1
+        fi
+        local agent_id="$2"
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)